test/e2e/network/network_tiers.go

```diff
@@ -134,7 +134,7 @@ var _ = SIGDescribe("Services [Feature:GCEAlphaFeature][Slow]", func() {
 		framework.ExpectEqual(svcTier, cloud.NetworkTierStandard)
 
 		// Wait until the ingress IP changes and verifies the LB.
-		ingressIP = waitAndVerifyLBWithTier(jig, ingressIP, createTimeout, lagTimeout)
+		_ = waitAndVerifyLBWithTier(jig, ingressIP, createTimeout, lagTimeout)
```

Comment:
can this be without the `_ = ` ?

Reply:
network_tiers.go:137:3: this value of ingressIP is never used (SA4006)

Maybe i can add above
//lint:ignore SA4006 avoid staticcheck
or
_ = waitAndVerifyLBWithTier(jig, ingressIP, createTimeout, lagTimeout)

do you have any other idea?Thanks

Reply:
can it be just:
```
waitAndVerifyLBWithTier(jig, ingressIP, createTimeout, lagTimeout)
```

instead of:
```
_ = waitAndVerifyLBWithTier(jig, ingressIP, createTimeout, lagTimeout)
```